iso.bbx

```diff
@@ -113,63 +133,92 @@
   {\item}%
 %
 
-%field formats
+% FIELD FORMATS
 
+% Format title and subtitle as is
 \DeclareFieldFormat*{title}{#1}
 \DeclareFieldFormat*{subtitle}{#1}
+
+% Format volume field with short localisation string
 \DeclareFieldFormat{volume}{\bibsstring{volume}\addnbspace#1}% volume of a book
+% Differentiate volume in journal
 \DeclareFieldFormat[article,periodical]{volume}{%
   \iftoggle{bbx:shortnum}
     {\mkbibbold{#1}}
     {\bibsstring{jourvol}\addnbspace#1}% volume of a journal
 }
+
+% Edition field could be:
+% a) numeral: use with localisation string
+% b) literal: capitalize given string
 \DeclareFieldFormat*{edition}{%
   \ifnumeral{#1}%
     {\mkbibordedition{#1}\addnbspace\bibsstring{edition}}%
     {\MakeCapital{#1}}%
 }%
+
+% Format pagination for page range
 \DeclareFieldFormat*{pages}{\mkmlpageprefix[bookpagination]{#1}}
+% Differentiate page range in journal
 \DeclareFieldFormat[article,periodical]{pages}{%
   \iftoggle{bbx:shortnum}
     {#1}
     {\mainsstring{pages}\addspace\printtext{#1}}%
```

Comment:
```latex
    {\mainsstring{pages}\addspace\printtext{#1}}%
```
is a bit odd.

There should usually be no `\printtext` in a field format declaration (I think they can mess up punctuation tracking and are in general not needed anyway).

Usually the pages format uses something like `\mkpageprefix[bookpagination]{#1}`  instead of `\mainsstring{pages}\addspace\printtext{#1}`. I guess what we really want here is a `\mkpageprefix` clone that uses `\mainsstring` instead of `\bibstring`.

Reply:
I think we have already prepared this, and it is the same as for
```
\DeclareFieldFormat*{pages}{\mkmlpageprefix[bookpagination]{#1}}
```
We only need to differentiate the switch of short numeration, otherwise it is the same.